packages/docs/docs/circuits-api/base-proof.md

```diff
@@ -41,7 +41,7 @@ constructor(
 
```

Comment:
on line 76:
- remove comma after 'constructor' and replace it after 'manually'
- should `Prover` and `prover` both be capitalized or lowercase?

Reply:
`Prover` is an interface: https://github.com/Unirep/Unirep/blob/81bb7605b3f7fd215ca15f05c9cdc6c119841a83/packages/circuits/src/circuits.ts#L19
and `prover` is an object with the `Prover` interface.
Is the description correct here?